### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by nnsW3**
Hello
I fixed several minor typos
Hope it helps
Br, Elias.

### CODE REVIEW

File: docs/book/src/reference/keywords.md [hunk 1]
```diff
@@ -11,6 +11,8 @@ traits, etc.
 The following is a list of keywords currently in use, with their
 functionality described.
 
+
+- `as` - rename items in `use` statements, e.g., `use type::a as alias_name`
```
Comment (xunilrj) [inline docs/book/src/reference/keywords.md:15]: Isn't this one duplicating this item?

Comment (nnsW3) [reply docs/book/src/reference/keywords.md:15]: yes it was, i fixed it

**Comment by xunilrj** [comment]
@nnsW3 CI failed because of some lint error.

**Comment by sdankel** [comment]
Fixed in https://github.com/FuelLabs/sway/pull/5975

**Pull request closed without merging**